package.json

```diff
@@ -79,29 +78,29 @@
     "gulp-rename": "1.2.2",
     "gulp-run": "1.7.1",
     "gulp-sourcemaps": "^2.4.1",
-    "gulp-spawn-mocha": "3.3.1",
+    "gulp-spawn-mocha": "4.0.1",
     "gulp-symlink": "2.1.4",
     "gulp-unzip": "0.2.0",
     "gulp-zip": "4.0.0",
-    "istanbul": "1.1.0-alpha.1",
+    "istanbul": "0.4.5",
```

Comment:
Downgrading istanbul leads to 
``` shell
No coverage information was collected, exit without writing coverage information
```

Reply:
I had updated instanbul with the stable release, but yes the coverage was failing. Reverted back to `"istanbul": "1.1.0-alpha.1"`.